indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -695,9 +695,12 @@ public void doRun()
           for (File file : toMerge) {
             indexes.add(HadoopDruidIndexerConfig.INDEX_IO.loadIndex(file));
           }
+
+          log.info("starting merge of intermediate persisted segments...");
```

Comment:
Ok, can the formatting be made more consistent with other logging messages. like simply

`Starting merge of intermediate persisted segments`